tests/SelfTest/UsageTests/ToStringGeneral.tests.cpp

```diff
@@ -27,9 +27,8 @@ TEST_CASE( "Character pretty printing" ){
         char space = ' ';
         CHECK(space == ' ');
         char chars[] = {'a', 'z', 'A', 'Z'};
-        for (int i = 0; i < 4; ++i){
-            char c = chars[i];
-            REQUIRE(c == chars[i]);
+        for (char c : chars){
+            REQUIRE(c == c);
         }
     }
```

Comment:
These tests are all wrong, I'll fix them in main and you can drop the commit.

Reply:
Dropped.